src/org/usfirst/frc4904/autonly/LeftSideTime.java

```diff
@@ -1,9 +1,13 @@
 package org.usfirst.frc4904.autonly;
 
 import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.Noop;
+import org.usfirst.frc4904.standard.commands.RunIf;
 
 public class LeftSideTime extends SideSwitchTime {
 	public LeftSideTime () {
-		super(RobotMap.gamefield.ourSwitch.left);
-	}
+    	addSequential(new RunIf(new Noop(), () -> {
```

Comment:
Again, no Noop.

Reply:
There are a few other places this happens but I was too lazy to go around and add comments on each one. Just imagine I did and change them 👌